docs/data/material/components/buttons/buttons.md

```diff
@@ -210,4 +228,22 @@ import Button from '@mui/material-next/Button';
 
 {{"demo": "ButtonMaterialYouPlayground.js", "hideToolbar": true, "bg": "playground"}}
 
-To learn more about Material UI's MD3 implementation, visit the [Material 3 Components documentation](/material-ui/guides/material-3-components/).
+To learn more about Material UI's MD3 implementation, visit the [Material 3 Components documentation](/material-ui/guides/material-3-components/).
+
+## Anatomy
+
+The Material UI Button component is composed of a single root `<button>` element with a `<span>` nested inside that houses the touch ripple effect.
+Optional [start and end icons](#start-and-end-icons) are rendered inside of `<span>` elements.
+
+```html
+<button class="MuiButtonBase-root MuiButton-root">
+  <span class="MuiButton-startIcon">
+    <svg class="MuiSvgIcon-root"></svg>
+  </span>
+  <!-- Button contents here -->
+  <span class="MuiButton-endIcon">
+    <svg class="MuiSvgIcon-root"></svg>
+  </span>
+  <span class="MuiTouchRipple-root"></span>
+</button>
+```
```

Comment:
I think we should reverse the links, I tend to click on Base UI thinking it's the Button unstyled, but it's the homepage.
 
![Screenshot 2024-01-30 at 18 16 22](https://github.com/mui/material-ui/assets/3165635/4d3d7354-5480-4e47-95c8-d6cca17fc248)

Reply:
Or remove the "Base UI" link entirely?!

Reply:
Alright, done #40875.